### PR TITLE
[sequelize] Update addColumn and removeColumn

### PR DESCRIPTION
**Pull request by nikola-vidic**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://sequelize.readthedocs.io/en/latest/docs/migrations/
{tableName:string, schema:string} can be used instead of tableName:string

- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/sequelize/sequelize-tests.ts [hunk 1]
```diff
@@ -1144,6 +1144,8 @@ queryInterface.createTable( 'users', { id : { type : Sequelize.INTEGER, primaryK
 queryInterface.createTable( 'level', { id : { type : Sequelize.INTEGER, primaryKey : true, autoIncrement : true } } );
 queryInterface.addColumn( 'users', 'someEnum', Sequelize.ENUM( 'value1', 'value2', 'value3' ) );
 queryInterface.addColumn( 'users', 'so', { type : Sequelize.ENUM, values : ['value1', 'value2', 'value3'] } );
+queryInterface.addColumn({tableName:'users', schema:'test'}, 'enum',{ type : Sequelize.ENUM, values : ['value1', 'value2', 'value3'] });
+queryInterface.removeColumn('users','so');
```
Comment (oktapodia) [inline types/sequelize/sequelize-tests.ts]: Can you test your new `removeColumn` implementation either, please?

**Review by oktapodia — CHANGES_REQUESTED** [review]

**Review by TitaneBoy — CHANGES_REQUESTED** [review]
image


Where does come from the {tableName?: string, schema?: string} type for addColumn and removeColumn ? Can't see anything about it in [queryInterface documentation](http://docs.sequelizejs.com/class/lib/query-interface.js~QueryInterface.html#instance-method-addColumn) neither in query-interface.js file... Can you provide us some references about these type changes ? Thanks

File: types/sequelize/sequelize-tests.ts (hunk 1 above)
Comment (nidzov) [reply types/sequelize/sequelize-tests.ts]: Yeah of course.

**Comment by nidzov** [comment]
Hey, 
if you look at http://sequelize.readthedocs.io/en/latest/docs/migrations/ and the usage of the methods. There are example usages of those two methods: 

```
// or with an explicit schema:

queryInterface.addColumn({
    tableName: 'Person',
    schema: 'public'
  },
  'signature',
  Sequelize.STRING
)

```
I don't know why it isn't in the documentations.

**Comment by oktapodia** [comment]
@TitaneBoy this is badly documented but we can found the same stuff into the code (deeper)

[Here](https://github.com/sequelize/sequelize/blob/master/lib/query-interface.js#L525) then [here](https://github.com/sequelize/sequelize/blob/master/lib/dialects/mysql/query-generator.js#L109), then [here](https://github.com/sequelize/sequelize/blob/master/lib/dialects/abstract/query-generator.js#L665) and finally [here](https://github.com/sequelize/sequelize/blob/master/lib/dialects/abstract/query-generator.js#L674)

From here

**Comment by TitaneBoy** [comment]
@oktapodia .. Thanks for the references...I didn't read all the sequelize code and I've just tried to see information in the code functions. If tests passes, then it will be good for me..

@nidzov Good catch...Thanks for your reference... Looks great for me.

**Review by TitaneBoy — APPROVED** [review]

**Review by oktapodia — APPROVED** [review]

**Pull request merged**